### PR TITLE
LPD-29842 | master

### PR DESCRIPTION
**Pull request by marcosapmf**
Jira ticket: https://liferay.atlassian.net/browse/LPD-29842

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/event-analysis/components/event-analysis-editor/event-analysis-builder/attribute-breakdown-dropdown/index.tsx [hunk 1]
```diff
@@ -148,20 +211,32 @@ const AttributeBreakdownDropdown: React.FC<IAttributeBreakdownDropdownProps> = (
...
-										const modifieldEventAttributeDefinitions = attribute
-											? eventAttributeDefinitions.map(
-													eventAttributeDefinition => {
-														if (
-															attribute.id ===
-															eventAttributeDefinition.id
-														) {
-															return attribute;
-														}
+										let modifieldEventAttributeDefinitions = [];
```
Comment (rachaelkoestartyo) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/event-analysis/components/event-analysis-editor/event-analysis-builder/attribute-breakdown-dropdown/index.tsx:214]: typo?

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:forward

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
blocked by https://github.com/liferay-frontend/liferay-portal/pull/4340

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
The format source issues are not related to this pull. Manually forwarding to https://github.com/brianchandotcom/liferay-portal/pull/152700

**Pull request closed without merging**